packages/cli/src/help/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Help } from '@oclif/core'
+import { BaseCommand } from '@preevy/cli-common'
+
+export default class MyHelpClass extends Help {
+  protected async showRootHelp(): Promise<void> {
+    await super.showRootHelp()
+    this.log(this.formatFlag())
+  }
+
+  protected formatFlag():string {
+    type Flag = {
+      name: string
+      description: string
+    };
+
+    const flags: Flag[] = []
+
+    for (const flagName of Object.keys(BaseCommand.baseFlags)) {
+      if (BaseCommand.baseFlags.hasOwnProperty.call(BaseCommand.baseFlags, flagName)) {
```

Comment:
why is this filter needed? there is no prototypical inheritance in `baseFlags` and we don't need to support it.